modules/evm/src/lib.rs

```diff
@@ -644,6 +652,7 @@ pub mod module {
 			Ok(().into())
 		}
 
+		/// Set the code of a contract at a given address.
```

Comment:
We want have more docs for all the extrinsics including the input parameters, and constraints.